images/dive/config/template.apko.yaml

```diff
@@ -1,6 +1,6 @@
 contents:
   packages:
-    - dive
```

Comment:
There seem to be a bunch of these changes that are whitespace or ordering. Can you format or revert any of these, to avoid churn?

Reply:
They also all coincide with renames, so it isn't zero churn.  I think these are largely from bringing back the packages manually :-/

Reply:
tl;dr it wouldn't change the "153 files changed" :)